rfcs/20210519-single-pass-bnorm/README.md

```diff
@@ -0,0 +1,126 @@
+# Proposal to calculate mean & variance in batchnorm in single pass
+
+## Requester
+GPU Enabling team
```

Comment:
Usually requester is a team working on a framework or an application that requests performance optimizations or new functionality to help them to achieve/get closer to performance targets set for their application or framework.